website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "React Native for macOS"
+author: Kiki Saintonge
+authorURL: http://twitter.com/KikiSaintonge
+description: "We're on React Native core version 0.62 with new community modules, native module support across multiple platforms, and of course our official public release of React Native for macOS."
+is_blog: true
+published_date: May 17, 2020
+---
+
+We're on React Native core version 0.62 with new community modules, native module support across multiple platforms, and of course our official public release of React Native for macOS.
+
+<!--truncate-->
+
+## Version 0.62-preview Released
+
+As promised in our previous milestone post, we're chugging along and keeping as up-to-date with the latest React Native core version as we can. That means for this latest milestone release we are officially announcing our [0.62-preview of React Native for Windows](https://www.npmjs.com/package/react-native-windows/v/0.62.0-preview.6)!
+
+As usual this means that our getting started guides and general docs will also be updated to reflect these changes.
+
+## React Native for macOS
+
+![react-native-mac](assets/eloy_rn4m_preview_full.png)
+
+One of our biggest announcements recently is our addition and full support of the React Native for macOS extension! We are super excited to be developing React Native support for macOS along side our Windows effort.
+
+Although still a fork, we have an official [npm package out for React Native for macOS](https://www.npmjs.com/package/react-native-macos) and a [repo available](https://github.com/microsoft/react-native-macos) to the public to follow and contribute to - we hope to see some really cool React Native apps out there in the future that can now truly run everywhere.
+
+## New Module Support!
+
+We've begun addressing the Windows community module support gap - starting with two contributions from us! The [**WebView**](https://github.com/react-native-community/react-native-webview) and [**Camera**](https://github.com/react-native-community/react-native-camera) modules.

```

Comment:
I think we have added more than Camera and WebView. [Picker](https://github.com/react-native-community/react-native-picker/pull/58), [AsyncStorage](https://github.com/react-native-community/async-storage/pull/327), [NetInfo](https://github.com/react-native-community/react-native-netinfo/pull/349) and [DateTimePicker](https://github.com/react-native-community/datetimepicker/pull/157) have all landed for Windows. 

WebView, NetInfo and AsyncStorage also have macOS support from us.

Reply:
added these module callouts as well.